vsdkx/addon/group/Agglomerative.py

```diff
@@ -0,0 +1,47 @@
+from sklearn.cluster import AgglomerativeClustering
+from vsdkx.addon.group.DBSCAN import GroupProcessor
+
+
+class AgglomerativeGroupProcessor(GroupProcessor):
+    """
+    Clusters the detected bounding boxes into groups, based on the distance
+    between the bounding boxes:
+    1. Clusters bounding boxes
+    2. Separates clusters into groups
+    3. Creates one bounding box per group
+
+    Attributes:
+        distance_threshold (int | float): Distance threshold required by
+        the clustering algorithm
+    """
+
+    def __init__(self, addon_config: dict, model_settings: dict,
+                 model_config: dict, drawing_config: dict):
+        super().__init__(addon_config, model_settings, model_config,
+                         drawing_config)
+        self.distance_threshold = 0.2
```

Comment:
We should have a `settings.yaml` file here and get the value for `distance_threshold` from there instead of having it hardcoded in the code.

Reply:
That's incompatible with our current architecture. We should decide, either it's needed to configure that with the settings received from outside and set default value there, or hardcode it in the code code.
These are packages, not usual repositories, code is not meant to be changed often here and config is not meant to be here at all.

Reply:
is this resolved?